### PR TITLE
Port python task

### PR DESCRIPTION
**Pull request by Abhay-dot**

### CODE REVIEW

**Review by rhl-bthr — CHANGES_REQUESTED** [review]
There are a lot of changes for whitespaces and indentation which make it hard to review. If those are for setting PEP8 standards, please add them in a separate commit

File: sugargame/canvas.py [hunk 1]
```diff
@@ -4,18 +4,17 @@
...
-import event
+import sugargame.event as event
 
 CANVAS = None
 
 
 class PygameCanvas(Gtk.EventBox):
-    def __init__(self, activity, pointer_hint=True,
-                 main=None, modules=[pygame]):
-        GObject.GObject.__init__(self)
```
Comment (rhl-bthr) [inline sugargame/canvas.py:15]: Since `GObject` is no longer being used, we can remove it's import

Comment (Abhay-dot) [reply sugargame/canvas.py:15]: ok removed it, thanks for the suggestion

Comment (rhl-bthr) [reply sugargame/canvas.py:15]: Let's also ensure that the `sugargame` files in the PR conform to the code in https://github.com/sugarlabs/sugargame. If it does not, and there's a good reason for that - then we can make appropriate changes in https://github.com/sugarlabs/sugargame

**Review by rhl-bthr — CHANGES_REQUESTED** [review]
Great work. Let's separate the whitespace changes in a different commit so that the Python 3 changes are easy to review and store. I see that you are using the GitHub UI to make edits to the files. I'd recommend you to use the `git` CLI to make changes and push them

**Review by rhl-bthr — APPROVED** [review]

**Comment by rhl-bthr** [comment]
Reviewed, not tested. Looks good.
Some whitespace changes are still present though.

**Pull request merged**